replica/tablets.cc

```diff
@@ -636,6 +643,21 @@ tablet_id process_one_row(replica::database* db, table_id table, tablet_map& map
     tablet_logger.debug("Set sstables_repaired_at={} table={} tablet={}", sstables_repaired_at, table, tid);
     map.set_tablet(tid, tablet_info{std::move(tablet_replicas), repair_time, repair_task_info, migration_task_info, sstables_repaired_at});
 
+    if (db->features().strongly_consistent_tables) {
+        if (row.has("raft_group_id")) {
```

Comment:
Might as well drop the outer if. If the feature is not enabled, the row won't have that column.